recipes/libx265/all/conanfile.py

```diff
@@ -49,22 +49,23 @@ def config_options(self):
             del self.options.fPIC
         if self.settings.os != "Linux":
             del self.options.with_numa
+        # FIXME: Disable assembly by default if host is arm and compiler apple-clang for the moment.
+        # Indeed, apple-clang is not able to understand some asm instructions of libx265
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch:
```

Comment:
do you have an error? this might not be desired for many people to have assembly optimization disabled...

Reply:
Sure, it doesn't compile. But it's better to compile without assembly optimization than nothing? Currently recipe raises if not x86 or x86_64...